src/arch/xtensa/include/arch/offset-defs.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2017, Intel Corporation
```

Comment:
Copyright year should be 2019 (as it's a new file)

Reply:
Fixed